src/modules/query/reducer.ts

```diff
@@ -45,7 +44,7 @@ export const initialState: ReducerState = {
   eventCollection: null,
   targetProperty: null,
   percentile: undefined,
-  timezone: DEFAULT_TIMEZONE,
+  timezone: null,
```

Comment:
As it's right now `null` if someone do not provide default timezone - the all queries will not work as timezone will be no set.